clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -721,6 +730,11 @@ protected Node curNode() {
             return curNode;
         }
 
+        final void incrementRetryBackoff(Call failedCall, long now) {
```

Comment:
For DescribeConsumerGroup and ListOffsets, AdminClient will construct a chain of requests. For example, DescribeConsumerGroup will send a FindCoordinatorRequest and then a DescribeGroupRequest (Denote them as F and D). Let's consider this case:
F (OK, tries = 0) -> D (Error, tries = 0) -> F (OK, tries = 1) -> D (Error, tries = 1) -> F (OK, tries = 2) -> D (OK, tries = 2)
Either F and D need to be aware of the number of attempts of the previously failed request to set their own attempts properly. Since F and D are separate objects and does not share any memory region, we can probably only to pass the failed call instance for constructing their status properly.